backend/src/main/java/wooteco/prolog/studyLogDocument/domain/StudyLogDocumentRepository.java

```diff
@@ -0,0 +1,13 @@
+package wooteco.prolog.studyLogDocument.domain;
+
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.elasticsearch.annotations.Query;
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
+
+public interface StudyLogDocumentRepository extends
+    ElasticsearchRepository<StudyLogDocument, Long> {
+
+    @Query("{\"query_string\": {\"fields\": [\"title\",\"content\"], \"query\": \"*?0*\"}}")
```

Comment:
```suggestion
    @Query("{\"query_string\": {\"fields\": [\"title\",\"content\"], \"query\": \"*:keyword*\"}}")
```
쿼리 신기하게 생겼네요...!! JPA책에서는 이 방식을 추천하고 있습니다..!

Reply:
이 것도 되나 한번 확인해볼게요!!! 감삼돵 👍🏻👍🏻

Reply:
elasitcsearch는 ?0, ?1 이렇게만 된다네요 ㅠㅅ ㅠ

Reply:
헐 대박 리얼루????????????????????????????????
이거 엘라스틱서치 PR 각이다